diesel/src/pg/types/ranges.rs

```diff
@@ -0,0 +1,125 @@
+
```

Comment:
Make sure to `cargo install rustfmt-nightly` which is the version with all the latest changes. (Or better yet, `cargo install --git https://github.com/rust-lang-nursery/rustfmt` directly)